### ISSUE TITLE
📱 直近のイベント情報をどう表示するか決める

### ISSUE THREAD

**Issue by yasulab** [coderdojo-japan/coderdojo.jp#410]
cf. [直近の CoderDojo 開催情報を表示したい #258](https://github.com/coderdojo-japan/coderdojo.jp/issues/258)

> 例えば統計情報がまとめられているデータベースを使って、直近の CoderDojo 開催情報を表示できるようにしようというプロジェクトがあって、「(イベントデータはなんとか収集できたという前提で) 収集した直近のイベント情報をどう表示するか？」とかはまだあまり具体化てきていない印象ですね 💭
> 
> 保護者や親御さんに見られることが多いからか、モバイル端末からアクセスされることが多いので、より mobile-friendly な状態にできたらなぁとは考えています 📱👀✨

![image](https://user-images.githubusercontent.com/155807/55388775-d338f680-556e-11e9-964e-fd2616f677f6.png)

_引用元: https://www.facebook.com/groups/coderdojo.jp/permalink/2115809078532480/?comment_id=2115890365191018&reply_comment_id=2116100811836640&comment_tracking=%7B%22tn%22%3A%22R%22%7D_

**Comment by RyoAbe** [comment]
@yasulab 質問です！データの収集は、こちらの issue  https://github.com/coderdojo-japan/coderdojo.jp/issues/258 で対応中ということで合ってます？
あと、どういったSchemeでデータが格納されるかの決定は、これからって感じですかね？

**Comment by yasulab** [comment]
@RyoAbe そうですね! 🙆 データ収集はまだ対応中で、具体的には Scheme を作るところ (なのでまだだいぶ初期段階) といった状況です >< 💦
